db/migrate/20200310140717_create_family.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFamily < ActiveRecord::Migration[6.0]
+  def change
+    create_table :families do |t|
+      t.integer :owner
+      t.integer :member_type
```

Comment:
Why do you create fields and remove them in the next migration? Please contain all needed DB changes in 1 migration.
`rake db:rollback` for each migration, then delete them, and create\modify only one migration file to contain all changes

Reply:
putted all my changes in 1 migration